changelogs/unreleased/gh-8333-fix-ignoring-tzoffset-in-parse.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/lua/datetime
+
+* Fixed a bug that caused `datetime.parse()` ignores `tzoffset`
+  option if custom format is used (gh-8333).
```

Comment:
```suggestion
* Fixed a bug that caused `datetime.parse()` ignore `tzoffset`
  option if a custom format was used (gh-8333).
```

Reply:
Fixed, thanks!

Reply:
@p7nov Pavel, please resolve comment if everything is ok from doc point of view.